packages/pharos/src/components/table/pharos-table.test.ts

```diff
@@ -0,0 +1,150 @@
+import { fixture, expect } from '@open-wc/testing';
+import type { TemplateResult } from 'lit';
+
+import { html } from 'lit/static-html.js';
+
+import type { PharosTable } from './pharos-table';
+import type { PharosLink } from '../link/pharos-link';
+import type { PharosPagination } from '../pagination/pharos-pagination';
+import type { PharosSelect } from '../select/pharos-select';
+
+describe('pharos-table', () => {
+  let component: PharosTable, componentWithPagination: PharosTable;
+  const columns = [
+    {
+      name: 'Item',
+      field: 'item',
+    },
+    {
+      name: 'Filename',
+      field: 'filename',
+    },
+  ];
+  const rowData = [
+    {
+      item: 1,
+      filename: '12345.jpg',
+      expired_date: '2020-1-1',
+      created_on: '2010-1-1',
+      university: 'University of Michigan',
+    },
+    {
+      item: 2,
+      filename: '123456.jpg',
+      expired_date: '2020-1-1',
+      created_on: '2010-1-1',
+      university: 'University of Michigan',
+    },
+  ];
+
+  beforeEach(async () => {
+    component = await fixture(html`
+      <test-pharos-table
+        .columns="${columns}"
+        .rowData="${rowData}"
+        .totalResults="${2}"
+        caption="test table"
+      >
+      </test-pharos-table>
+    `);
+
+    componentWithPagination = await fixture(html`
+      <test-pharos-table
+        .columns="${columns}"
+        .rowData="${rowData}"
+        .hidePagination="${false}"
+        .totalResults="${2}"
+        .pageSizeOptions="${[1, 2]}"
+        caption="test table with pagination"
+      >
+      </test-pharos-table>
+    `);
+  });
+
+  it('is accessible', async () => {
+    await expect(component).to.be.accessible();
+  });
+
```

Comment:
Might be nice to add a unit test like [this one](https://github.com/ithaka/pharos/blob/develop/packages/pharos/src/components/button/pharos-button.test.ts#L185) which verifies it throws an error when there is no caption?

Reply:
done!